README.md

```diff
@@ -55,22 +94,121 @@ through the Docker Compose
 [build](https://docs.docker.com/compose/reference/build/) command:
 
 ```
-# Build container images
 docker-compose build
 ```
 
-### Start
+## How to start the service
 
-Use Docker Compose to run service locally.
+The service can be ran using the convenience script:
 
-Additional Docker Compose files are provided for scenarios such as linking to other running services.
+```
+./scripts/start
+```
+
+Or use Docker Compose to run service locally.
 
+```
+docker-compose start
+```
+
+Additional Docker Compose files are provided for scenarios such as linking to other running services.
 Link to other services:
+
 ```
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
-## Test structure
+> Note: This service depends on the 'ffc-pay-processing' service to be able to run correctly.
+
+## How to get output
+
+There are several different possible outputs:
+
+1. **To access the web front-end for the service**
+
+**Pre-requisite:** Start the ffc-pay-processing service (as described in the README.md of it's repository.
```

Comment:
Suggest putting all references to `ffc-pay-processing` in back ticks ` so easier to read.

Reply:
Could also make `README.md` a link too